client/include/frontier_client/frontier.h

```diff
@@ -26,7 +26,6 @@
 typedef unsigned long FrontierChannel;
 typedef void FrontierRSBlob;
 
-
```

Comment:
Was this intentional?  I typically leave 2 newlines after `typedef`s.

(It's fine if it was on purpose - just not clear from the coding style.)

Reply:
I think it was accidental and it probably happened because at first I had put the new structs up there.  However there's no other double blank line in the file so I think I'd rather leave it as it is at this point.